src/configureWorkspace/configureDotNetCore.ts

```diff
@@ -462,3 +464,11 @@ export async function scaffoldNetCore(context: ScaffolderContext): Promise<Scaff
 
     return files;
 }
+
+export function ensureDotNetCoreDependencies(workspaceFolder: WorkspaceFolder, context: IActionContext): void {
+    if (!hasTask('build', workspaceFolder)) {
+        context.errorHandling.suppressReportIssue = true;
+        const message = localize('vscode-docker.configureDotNetCore.missingDependencies', 'Dependency build task is missing. Please generate build task by running \'.NET: Generate Assets for Build an Debug\' before running this command');
```

Comment:
The "dependency" word here might be understood as "we are missing a build task for a dependency", which is not the case. Suggest saying simply "A build task is missing... " (rest of the error message unchanged).